src/test/java/com/microsoft/graph/content/BatchRequestContentTest.java

```diff
@@ -1,269 +0,0 @@
-package com.microsoft.graph.content;
```

Comment:
I'd also keep the unit tests related to tasks (batch, large file upload...)

Reply:
added batch test, large file upload tests never existed.